experimental.d.ts

```diff
@@ -23,7 +23,7 @@ export const echo: Echo
 interface Retry {
   (pieces: TemplateStringsArray, ...args: any[]): Promise<ProcessOutput>
 }
-export const retry: (count: number) => Retry
+export const retry: <C extends number = 5, D extends number = 0>(count?: C, delay?: D) => Retry
```

Comment:
Why extend 5?)

Reply:
```ts
export const retry: (count: number = 5) => Retry
```
raises an error `TS2371: A parameter initializer is only allowed in a function or constructor implementation`:
So the only way to add a default arg value to fn type declaration is via generics (afair).

Reply:
nvm, reverted